lib/hydra/pcdm/deep_member_iterator.rb

```diff
@@ -0,0 +1,25 @@
+module Hydra::PCDM
+  class DeepMemberIterator
+    include Enumerable
+    attr_reader :record
+    def initialize(record)
+      @record = HasMembers.new(record)
+    end
+
+    def each
+      record.members.each do |member|
+        yield member
+      end
+      record.members.each do |member|
```

Comment:
It pains me a bit to see two identical iterators so close to one another but I get why you're doing this (spit out all members before iterating more deeply), so no bigs.